orchestration/dagster_orchestration/hca_orchestration/contrib/argo_workflows.py

```diff
@@ -29,23 +31,20 @@ def __call__(
     ) -> V1alpha1WorkflowList: ...
 
 
+@dataclass
```

Comment:
huh TIL about dataclasses, thanks steve

Reply:
i only learned about dataclasses + the cached_property library recently myself, i'm just happy they can save us some code.